v2/components/schemas/Notice.yaml

```
@@ -0,0 +1,62 @@
+# SPDX-License-Identifier: GPL-3.0
+# SPDX-FileCopyrightText: Canonical Ltd
+
+type: object
+description: A notice recorded by snapd, such as a warning or change update.
+properties:
+  id:
+    type: string
+    description: The unique ID of the notice.
+    example: "67"
+  user-id:
+    type: string
+    nullable: true
+    description: The UID of the user who may view the notice, or null if public.
+    example: null
+  type:
+    type: string
+    description: The type of the notice.
+    enum:
+      - change-update
+      - warning
+      - refresh-inhibit
+      - snap-run-inhibit
+  key:
+    type: string
+    description: |-
+      A type-specific identifier for the notice, often corresponding to a change ID,
+      can also be the name of the snap associated with the notice.
```

I think we should either list examples for all types, or say "For example" and specify which types those examples cover:

```suggestion
      A type-specific identifier for the notice. For example, "change-update" notices
      have keys corresponding to their change ID, while "snap-run-inhibit" notices use
      the snap name as the key.
```

---

I think this is the same as an above comment? @olivercalder  if so it has been addressed now! If not, could we perhaps sync later today and discuss?